app/controllers/submit.js

```diff
@@ -279,9 +284,30 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
         const modelDOI = this.get('model.doi');
         return (basicsDOI || modelDOI) && basicsDOI !== modelDOI;
     }),
-    basicsChanged: Ember.computed('tagsChanged', 'abstractChanged', 'doiChanged', function() {
+    basicsLicense: Ember.computed('model', function() {
+        let record = this.get('model.licenseRecord');
+        let license = this.get('model.license');
+        return {
+            year: record ? record.year : null,
+            copyrightHolders: record && record.copyright_holders ? record.copyright_holders.join(',') : null,
+            licenseType: license || null
+        };
+    }),
+    licenseChanged: Ember.computed('model.license', 'model.licenseRecord', 'basicsLicense.year', 'basicsLicense.copyrightHolders', 'basicsLicense.licenseType', function() {
+        let record = this.get('model.licenseRecord');
+        let license = this.get('model.license') || null;
+        let changed = false;
+        if (record) {
+            changed = (record.year !== this.get('basicsLicense.year')) || (this.get('basicsLicense.copyrightHolders') !== record.copyright_holders.join(','));
```

Comment:
When you load a preprint in Edit Mode, changed can be true here, which is incorrect.  This means licenseChanged is true, and thus basicsChanged is true.  It looks like `(this.get('basicsLicense.copyrightHolders') !== record.copyright_holders.join(','))` is comparing an empty array to an empty string and saying they are not equal.
![screen shot 2016-12-01 at 5 21 30 am](https://cloud.githubusercontent.com/assets/9755598/20790422/3e30f0f2-b786-11e6-94e7-007696cc129e.png)